common/predictive-text/worker/index.ts

```diff
@@ -33,56 +116,84 @@ class LMLayerWorker {
    * so that this can be tested **outside of a Worker**.
    */
   private _postMessage: PostMessage;
-  private _importScripts: ImportScripts;
 
   constructor(options = {
     postMessage: null,
-    importScripts: null
   }) {
     this._postMessage = options.postMessage || postMessage;
-    this._importScripts = options.importScripts || importScripts;
   }
 
   /**
    * A function that can be set as self.onmessage (the Worker
    * message handler).
    * NOTE! You must bind it to a specific instance, e.g.:
-   * 
+   *
    *   // Do this!
    *   self.onmessage = worker.onMessage.bind(worker);
-   * 
+   *
    * Incorrect:
-   * 
+   *
    *   // Don't do this!
    *   self.onmessage = worker.onMessage;
-   * 
+   *
    * See: .install();
    */
   onMessage(event: MessageEvent) {
     const {message} = event.data;
+    // We must have gotten a message!
     if (!message) {
       throw new Error(`Missing required 'message' attribute: ${event.data}`)
     }
 
-    // Load the model.
-    let model = this._importScripts(event.data.model);
+    // ...that message must have been 'initialize'!
+    if (message !== 'initialize') {
```

Comment:
I assume this is a temporary thing, right?  It'd probably be better as `if(!this.isInitialized && message !== 'initialize')` in the long-term, with other options afterward.

Reply:
This is indeed temporary. In the prototype, I implemented [state pattern](https://en.wikipedia.org/wiki/State_pattern) instead of using an auxiliary `isInitialized` property; either way, this is _not_ the final code for `onMessage()`.